src/components/atoms/Icons/Wallet.tsx

```diff
@@ -37,3 +38,7 @@ export const MetamaskWalletDisabledIcon: React.FC<SvgIconProps> = props => (
 export const TrustWalletDisabledIcon: React.FC<SvgIconProps> = props => (
   <SvgIcon component={TrustDisabled} viewBox="0 0 32 32" {...props} />
 );
+
+export const CommontWalletIcon: React.FC<SvgIconProps> = props => (
```

Comment:
This one as well

Reply:
resole in https://github.com/myriadsocial/myriad-web/pull/1515/commits/c1d083d95c731d2f4b2b5a0fff1f7acc652debed